graphistry/feature_utils.py

```diff
@@ -47,6 +47,16 @@
         SuperVectorizer = Any
         GapEncoder = Any
         SimilarityEncoder = Any
+    try:
```

Comment:
sanity check: does cuCat do lazy cudf etc. imports?

Reply:
cu-cat is all cupy, no cudf import needed. 
i see now that is how we cant to do gpu all the way thru, but it shouldnt be hard to adapt since the speed/scale issue is solved

Reply:
`cupy` import is also slow i think, my point is to ensure `cu_cat` import is lazy here as i'm guessing the `cu_cat` package is slow to load

Reply:
oh sorry never mind... this is during type checking, that's fine :)